go-controller/pkg/metrics/metrics.go

```diff
@@ -411,3 +411,18 @@ func RegisterOvnMetrics(clientset kubernetes.Interface, k8sNodeName string) {
 	go RegisterOvnControllerMetrics()
 	go RegisterOvnNorthdMetrics(clientset, k8sNodeName)
 }
+
+// StartOVSMetricsServer runs the prometheus listener so that OVS metrics can be collected
+func StartOVSMetricsServer(bindAddress string) {
```

Comment:
no need for this function now - the metrics will be exposed by server instantiated by func `StartOVNMetricsServer`

Reply:
Good catch.